components/collections/index.vue

```diff
@@ -275,31 +314,42 @@ export default {
           path,
         })
         this.syncCollections()
-      }
-      else if (this.collectionsType.type === "team-collections") {
+      } else if (this.collectionsType.type === "team-collections") {
         if (this.collectionsType.selectedTeam.myRole != "VIEWER") {
-          team_utils.addChildCollection(this.$apollo, name, folder.id)
-          .then((data) => {
-            // Result
-            this.$toast.success(this.$t("folder_added"), {
-              icon: "done",
+          this.$apollo
+            .mutate({
+              mutation: gql`
+                mutation($childTitle: String!, $collectionID: String!) {
+                  createChildCollection(childTitle: $childTitle, collectionID: $collectionID) {
+                    id
+                  }
+                }
+              `,
+              // Parameters
+              variables: {
+                childTitle: name,
+                collectionID: folder.id,
+              },
             })
-            console.log(data)
-            this.updateTeamCollections();
-          })
-          .catch((error) => {
-            // Error
-            this.$toast.error(this.$t("error_occurred"), {
-              icon: "done",
+            .then((data) => {
+              // Result
+              this.$toast.success(this.$t("folder_added"), {
+                icon: "done",
+              })
+              console.log(data)
```

Comment:
remove console logs

Reply:
These console logs were there before as well, should I still remove them?